cli/aiconfig-editor/@types/ufetch.d.ts

```diff
@@ -0,0 +1,13 @@
+declare module "ufetch" {
```

Comment:
Might be worth publishing to @types or as part of ufetch at some point

Reply:
Yeah I haven't published ufetch in 7 years... will need to figure out how to do that again or just add to @types